.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,16 @@
-For bug reports, the following information can help speed up the process.
+For bug reports, the following information can help speed up the process. Please
+describe the bug that you have found and what you would expect to happen
+instead.
```

Comment:
Collapse these onto one line for GitHub's text editor.

Reply:
I don't think this makes a difference for GitHub's text editor, right?

I'd love if it would actually use these as three different lines, but based on my testing it seems like it merges all of these together into a single line anyways.

Reply:
I stand corrected, you are completely correct.

I've tested this by pushing to chrisduerr/alacritty master and I think it looks nice and sleek now.